GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -310,6 +311,7 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(RevisionGridControl.Command.RevisionFilter, Keys.Control | Keys.I),
                     Hk(RevisionGridControl.Command.SelectAsBaseToCompare, Keys.Control | Keys.L),
                     Hk(RevisionGridControl.Command.SelectCurrentRevision, Keys.Control | Keys.Shift | Keys.C),
+                    Hk(RevisionGridControl.Command.SelectNextForkPointAsDiffBase, Keys.Control | Keys.F),
```

Comment:
I want to start searches also when grid is focused.
Too bad `Crtl-Shift-P` is used already
Maybe `Crtl-Shift-K`

Reply:
`Ctrl+K` is free. (Also `Ctrl+J`. But even more arbitrary.)

Reply:
done and updated the description